protocol/generated.pb.go

```diff
@@ -1,14 +1,3 @@
-// Copyright 2017 Sourced Technologies SL
```

Comment:
Are we removing these comments then? If so, README should be updated.

Reply:
Yes, that's what I understood from the slack chat. README updated.